package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hmpo-template-mixins",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
I'd recommend doing `npm version patch` for this, instead of running it manually. That way it creates a release tag for you. This is fine really, but remember to tag `v0.1.1`.